File: src/main/java/org/lineageos/sleepaudio/ui/MainActivity.java
package org.lineageos.sleepaudio.ui;

import android.os.Bundle;
import com.android.settingslib.collapsingtoolbar.CollapsingToolbarBaseActivity;

public class MainActivity extends CollapsingToolbarBaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        
        if (savedInstanceState == null) {
            getFragmentManager().beginTransaction()
                .replace(com.android.settingslib.collapsingtoolbar.R.id.content_frame, new SleepAudioFragment())
                .commit();
        }
    }
}

File: app/src/main/kotlin/org/lineageos/sleepaudio/ui/MainActivity.kt
package org.lineageos.sleepaudio.ui

import android.os.Bundle
import com.android.settingslib.collapsingtoolbar.CollapsingToolbarBaseActivity
import com.android.settingslib.collapsingtoolbar.R as CollapsingR

class MainActivity : CollapsingToolbarBaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        
        if (savedInstanceState == null) {
            fragmentManager.beginTransaction()
                .replace(CollapsingR.id.content_frame, SleepAudioFragment())
                .commit()
        }
    }
}
